hack/e2e/run.sh

```diff
@@ -25,6 +25,7 @@ source "${BASE_DIR}"/util.sh
 
 DRIVER_NAME=${DRIVER_NAME:-aws-ebs-csi-driver}
 CONTAINER_NAME=${CONTAINER_NAME:-ebs-plugin}
+DRIVER_START_TIME_THRESHOLD=60
```

Comment:
one last thing, can you add a comment like `# seconds` here?

Reply:
Updated, added seconds in the name to make it clear.